src/logic/operate.js

```diff
@@ -0,0 +1,16 @@
+import { evaluate } from 'mathjs';
+
+const operate = (num1, num2, operation) => {
+  let op;
+  if (operation === 'X') {
+    op = '*';
+  } else if (operation === '÷') {
+    op = '/';
+  } else {
+    op = operation;
+  }
+
+  return evaluate(`${num1}${op}${num2}`);
+};
+
+export default operate;
```

Comment:
You been asked to use `big.js`, not `mathjs`, please change  this
![Screen Shot 2019-12-09 at 10 08 05 AM](https://user-images.githubusercontent.com/40712459/70421854-f7889380-1a6a-11ea-953e-c7195ab16e63.png)


- also, i can't find all operators here (example `+`)
![Screen Shot 2019-12-09 at 10 10 33 AM](https://user-images.githubusercontent.com/40712459/70421893-096a3680-1a6b-11ea-8b2b-e2a3b04ea648.png)

Reply:
I tested this and it was working for all operations. I only addressed these operations because in coding multiply is '*' instead 'X' and divide is '/'. Since all the other ones were the same thing I didn't need to change. I used mathjs because it recognize the operate symbol automatically inside the string and I don't need to call a specific method to subtract or multiply... By doing that I made my code better because is shorter.